gitlab/utils.py

```diff
@@ -55,15 +55,15 @@ def copy_dict(dest: Dict[str, Any], src: Dict[str, Any]) -> None:
             dest[k] = v
 
 
-def clean_str_id(id: str) -> str:
-    return id.replace("/", "%2F").replace("#", "%23")
+def clean_str_id(id):
```

Comment:
Can the type-hints not be removed?

Reply:
Uuuuh, absolutely yes, sorry. Fixing it now!